### ISSUE TITLE
How can I  get an hostname ?

### ISSUE THREAD

**Issue by loging1010** [martinlindhe/wmi_exporter#409]
like node_exporter's Metrics name node_uname_info.
for example:
node_uname_info{domainname="(none)",instance="172.18.0.79:9100",job="node-exporter",machine="x86_64",nodename="OPS-Zabbix-Server"}

**Comment by breed808** [comment]
@loging1010 do you need the hostname as a metric label? Usually this would be set on the Prometheus side as either the `instance` label (using DNS to map the hostname to an IP) or using static labels.

Static label example:
```yaml
- job_name: 'windows'
  static_configs:
    - targets: ['192.168.1.1']
      labels:
        instance: OPS-Zabbix-Server
```

DNS example:
```yaml
- job_name: 'windows'
  static_configs:
    - targets: ['OPS-Zabbix-Server'] # Resolves to 192.168.1.1
```

**Comment by alekssako** [comment]
@loging1010 or you can use `alias` too.
```
- job_name: 'windows'
  static_configs:
  - targets: ['192.168.1.1']
    labels:
      alias: OPS-Zabbix-Server
```

**Comment by loging1010** [comment]
> @loging1010 do you need the hostname as a metric label? Usually this would be set on the Prometheus side as either the `instance` label (using DNS to map the hostname to an IP) or using static labels.
> 
> Static label example:
> 
> ```yaml
> - job_name: 'windows'
>   static_configs:
>     - targets: ['192.168.1.1']
>       labels:
>         instance: OPS-Zabbix-Server
> ```
> 
> DNS example:
> 
> ```yaml
> - job_name: 'windows'
>   static_configs:
>     - targets: ['OPS-Zabbix-Server'] # Resolves to 192.168.1.1
> ```

thanks

**Comment by cobexer** [comment]
The node_exporter metric called `node_uname_info` is used in charts like https://grafana.com/grafana/dashboards/1860 to add a dropdown that shows the node name that doesn't depend on the method used to configure scrape targets.

In my case I can only use ip addresses to configure scrape targets and can't set any prometheus configuration.

Could you add such info? See also https://github.com/martinlindhe/wmi_exporter/issues/430
